dev/Pager/PrototypePager/Scripts/Int32ToDoubleConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Data;
+
+namespace MUXControlsTestApp.Scripts
+{
+    public class Int32ToDoubleConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            return System.Convert.ToDouble(value);
```

Comment:
>return System.Convert.ToDouble(value); [](start = 12, length = 38)

I know this was needed in your initial project, have you confirmed that it is still needed with this project configuration. I'm kind of surprised the xaml compiler can't figure this out on its own.

Reply:
I have not confirmed if its needed for this project config but I can look into it and will update this thread.

Reply:
After testing it out, it seems like the types need to be the same if I use a templated binding but if I use binding instead, it will figure it out on its own. I decided to remove most converters aside from one and replace some templated bindings with bindings instead